### PR TITLE
Added templates and contributing guilds

### PR DESCRIPTION
**Pull request by 7imbrook**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### tl;dr
+- Master branch is off limits for pushes, all code gets reviews by at least 2 members of the team members.
```
Comment (mwitte98) [inline CONTRIBUTING.md]: We don't need "members" twice 😊

**Comment by mwitte98** [comment]
LGTM

**Comment by 7imbrook** [comment]
LGTM

**Comment by 7imbrook** [comment]
Ok so thats a feature...

**Pull request merged**